Cargo.toml

```diff
@@ -12,7 +12,7 @@ readme = "README.md"
 license = "MIT"
 
 [dependencies]
-bdk = "^0.2"
+bdk = { version = "^0.3", default-features = false }
 bdk-macros = "^0.2"
```

Comment:
Should `bdk-macros` also be upgraded to `^0.3`?

Reply:
I just remembered the `bdk-macro` changes in https://github.com/bitcoindevkit/bdk/pull/174 just got merged so aren't in any crates.io version yet, so `^0.2` is fine.

Reply:
ACK 6075869 

I manually tested with a few simple `bdk-cli` commands.